polar-core/src/warnings.rs

```diff
@@ -5,6 +5,42 @@ use super::terms::*;
 
 use std::collections::{hash_map::Entry, HashMap};
 
+pub fn common_misspellings(t: &str) -> Option<String> {
+    let misspelled_type = match t {
```

Comment:
This would be a little less painful if it was:

```
let misspelled_type = match t { 
   "int" => "Number",
   ...
  _ => return None
};
Some(misspelled_type)